CMakeLists.txt

```diff
@@ -319,6 +319,8 @@ if(ENABLE_QT)
     set(ENABLE_QT_DBUS_INTEROP OFF)
 
     set(QT_REQUIRED_MODULES Core Gui Widgets Network LinguistTools)
+    set(Qt6 REQUIRED COMPONENTS Svg)
```

Comment:
The original change of adding `Svg` to `QT_REQUIRED_MODULES` was correct, current change isn't. Setting `Qt6` variable makes no difference because it's not used anywhere.

The reason it failed before was because QtSvg module isn't installed on CI. You can influence that by modifying actions.yml and changing https://github.com/transmission/transmission/blob/32531fe5efe0140abc4ed4c0d901abab78add514/.github/workflows/actions.yml#L263-L265 and https://github.com/transmission/transmission/blob/32531fe5efe0140abc4ed4c0d901abab78add514/.github/workflows/actions.yml#L535-L537 and https://github.com/transmission/transmission/blob/32531fe5efe0140abc4ed4c0d901abab78add514/.github/workflows/actions.yml#L608-L610

This would still leave builds running on Jenkins, but I'll fix it up once these changes are in.